PROTOCOL.md

```diff
@@ -424,6 +429,44 @@ enough to get you started. You can watch the data sent and received from
 the server on a regular connection, or look at the client source code
 for a full list of message types.
 
+**Action requests**
+
+These are how the client sends the player's decisions to the server. Most
+requests are sent with `REQUESTID`. `REQUESTID` is used to identify which
+action the request was intended for to protect from cases of trying to send
+multiple requests for the same action, which would otherwise effect future
+decisions. The first decision of a match has a `REQUESTID` of 1, and each
+successful decision increments the counter by 1.
+
+`|/team ORDER|REQUESTID`
+
+> Chooses the team order. Numbers not listed will be displaced to the back of the list.
+> For example `/team 25` sets the default team order to 251346.
```

Comment:
`/team 25` actually gives a team order of `253416`. I'm guessing that they swap positions or something, like so:

- The first number is 2, so 2 swaps with 1. The order is `213456`.
- The second number is 5, so 5 swaps with whatever number was in the second spot (1). The order is `253416`.

It doesn't actually matter anyhow, but you should just mention that `/team 25` sets the first two Pokemon to be numbers 2 and 5 and not mention how the rest of the team order goes.